src/UKHO.SAP.MockAPIService/Enums/TestCase.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace UKHO.SAP.MockAPIService.Enums
+{
+    public enum TestCase
+    {
+        Status500InternalServerError = 1
```

Comment:
```suggestion
        500InternalServerError = 1
```

Reply:
Identifiers cannot begin with a number.  A compilation error will occur.
updated to InternalServerError500